src/style.css

```diff
@@ -1 +1,185 @@
 /* add styles here */
+
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  display: block;
+  height: 16px;
+  width: 16px;
+  background-position-y: center;
+  background-position-x: center;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5),
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5),
+.stars--3 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(5),
+.stars--4 .stars__star:nth-child(5) {
+  background-image: url(./images/star.svg);
```

Comment:
It is still overcomplicated. Set the default background-image for all stars and then just change it for active stars